deploy/crds/v2v_v1alpha1_virtualmachineimport_crd.yaml

```diff
@@ -32,9 +32,66 @@ spec:
         metadata:
           type: object
         spec:
-          type: object
+          description: 'VirtualMachineImportSpec defines the desired state of VirtualMachineImport'
+          properties:
+            providerCredentialsSecret:
+              description: 'ProviderCredentialsSecret defines how a secret resource should be identified on kubevirt'
+              properties:
+                name:
+                  description: 'Name defines the name of the secret for source provider credentials'
+                  type: string
+                namespace:
+                  description: 'Namespace defines the namespace of the secret for source provider credentials'
+                  type: string
+            resourceMapping:
+              description: 'ResourceMapping defines how a ResourceMapping should be identified on kubevirt. VM Import will map resources of source provider
+                to kubevirt resources based on the mappings within ResourceMapping'
+              properties:
+                name:
+                  description: 'Name defines the name of the ResourceMapping to use'
+                  type: string
+                namespace:
+                  description: 'Namespace defines the namespace of the ResourceMapping to use'
+                  type: string
+            targetVmName:
+              description: 'TargetVmName defines the name of the target VM to be created'
+              type: string
+            startVm:
+              description: 'StartVm defines if the VM should be started at the end of the import process'
+              type: boolean
+            source:
+              description: 'VirtualMachineImportSourceSpec defines the definition of the source provider and mapping resources'
+              properties:
+                ovirt:
+                  description: 'VirtualMachineImportOvirtSourceSpec defines the definition of the VM in oVirt and the credentials to oVirt'
```

Comment:
` defines the definition of the VM in oVirt and the credentials to oVirt`
the credentials is not true anymore